data.sql

```diff
@@ -18,4 +18,98 @@ VALUES
   ('Angemon', '2005-06-12', 1, true, -45),
   ('Boarmon', '2005-06-07', 7, true, 20.4),
   ('Blossom', '1998-10-13', 3, true, 17),
-  ('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+  ('Ditto', '2022-05-14', 4, true, 22);
+
+/* Populate new tables with sample data */
+BEGIN WORK;
+
+INSERT INTO
+  owners(full_name, age)
+VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+INSERT INTO
+  species(name)
+VALUES
+  ('Pokemon'),
+  ('Digimon');
+
+
+
+/* MILESTONE--3 */
+
+/* Add corresponding data to species_id and owner_id column */
+BEGIN WORK;
+
+/* species_id data */
+UPDATE
+  animals
+SET
+  species_id = 2
+WHERE
+  name LIKE '%mon%';
+
+SAVEPOINT SP1;
+
+UPDATE
+  animals
+SET
+  species_id = 1
+WHERE
+  species_id IS NULL;
+
+COMMIT;
+
+/* owner_id data */
+BEGIN WORK
```

Comment:
- [ ] Kindly add a `;` at the end of this TRANSACTION. because posgres is trowing out an syntax error.

![image](https://user-images.githubusercontent.com/65068771/159829457-649ca1a3-066e-42cc-9ced-45d7460a8a85.png)

Reply:
Thanks for your review I didn't see that syntax error.

Reply:
You're welcome sir.